themes/default/README.md

```diff
@@ -0,0 +1,61 @@
+# Default Theme for Open-Oni
```

Comment:
I do like this README here, because I feel like as a dev I would notice it here before I went looking in the wiki, but I wonder if we want to include most of the information in the wiki and provide a link here to it?